site/el_EL.all.json

```diff
@@ -923,7 +923,7 @@
     "other": "Ακύρωση συνδρομής;"
   },
   "usersubscriptions_unsubscribe_modal_body": {
-    "other": "Αυτό θα ακυρώσει τη συνδρομή στο {{.Name}} σχέδιο. Θα εξακολουθείτε να μπορείτε να παρακολουθείτε το περιεχόμενο του προγράμματος έως και {{.Expiry}}."
+    "other": "Αυτό θα ακυρώσει τη συνδρομή στο {{.Name}}. Θα εξακολουθείτε να μπορείτε να παρακολουθείτε το περιεχόμενο μέχρι να λήξει {{.Λήξη}}."
```

Comment:
Gotta keep an eye out for those broken variables: `{{.Λήξη}}`!

Reply:
Fixed broken variables.